src/analyzing/CFG_build.mli

```diff
@@ -1 +1,17 @@
+(* Compute the control flow graph of a sequence of statements.
+ * This sequence of statements doesn't necessarily have to be
+ * a function body, so no implicit return analysis is done here.
+ *)
 val cfg_of_stmts : IL.stmt list -> IL.cfg
```

Comment:
Perhaps call this `cfg_of_top_level_stmts` ?

Reply:
Changing the name of this function will break inter-file while waiting for PR reviews and syncing between versions of the semgrep submodule. If possible, I'd like to make sure this PR integrates with inter-file well, then I can rename this function later.